AppCenter/AppCenter/Internals/Util/MSUtility+NSObject.m

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+#import "MSUtility+NSObject.h"
+
+/*
+ * Workaround for exporting symbols from category object files.
+ */
+NSString *MSUtilityObjectSelectorCategory;
+
+@implementation NSObject(PerformSelectorOnMainThreadMultipleArgs)
```

Comment:
```suggestion
@implementation MSUtility (PerformSelectorOnMainThreadMultipleArgs)
```

Reply:
When I did this I got an EXC_BAD_ADDRESS error, not sure why.